### ISSUE TITLE
A MISP galaxy with relationship

### ISSUE THREAD

**Issue by adulau** [cudeso/rsit-attck#9]
As MISP is now capable to deal with relationships between galaxy as as RSIT is growing, maybe it's time to convert RSIT as a galaxy? and create the relationships between the two (ATT&CK) and other galaxies if needed. Just an idea.

**Comment by cudeso** [comment]
Indeed, this might be a good idea. I'll need some time to think about the best approach ;-).

**Comment by cudeso** [comment]
@adulau : See https://github.com/cudeso/rsit-attck/commit/145f3cbb6f91f425d959684cd11b0bdc0f6737e4 and PR https://github.com/enisaeu/Reference-Security-Incident-Taxonomy-Task-Force/pull/88

I'll wait for the feedback of the working group before committing to the MISP galaxy repo.
